src/mediaelement.js

```diff
@@ -71,7 +71,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     getCurrentTime: function () {
-        return this.media.currentTime;
+        return this.media && this.media.currentTime;
```

Comment:
It wass called from audioprocess handler that was handled from clearPlayEnd that was called from pause.